c2corg_ui/static/js/map/map.js

```diff
@@ -145,6 +146,18 @@ app.MapController = function($scope, mapFeatureCollection, ngeoLocation,
   this.showRecenterTools;
 
   /**
+   * @type {boolean}
+   * @export
```

Comment:
I think we need it, else the directive attribute value (in the HTML) is not taken into account. Most likely because Closure renames the property where its name is fixed in the directive scope list above.